gen/sam/ewr_group_generator.py

```diff
@@ -64,5 +64,5 @@ def generate_ewr_group(
         generator_class = random.choice(generators)
         generator = generator_class(game, ground_object)
         generator.generate()
-        return list(generator.get_generated_group())
-    return []
+        return [generator.get_generated_group()]
```

Comment:
You don't need the list now. Other than that, ready to merge.

Reply:
Oops thanks.